releasenotes/notes/bosonic-logarithmic-mapper-4b1f24c4ca16cf8b.yaml

```
@@ -0,0 +1,17 @@
+---
+prelude: >
+    Up to now, Qiskit Nature could only map bosonic operators to qubit operators using
+    a linear encoding (see :class:`.BosonicLinearMapper`).
+features:
+  - |
+    This release introduces a new mapper for the BosonicOp, the BosonicLogarithmicMapper.
+    It is more efficient both in terms of the number of qubits required and the Pauli strings generated.
```

```suggestion
    It is more efficient both in terms of the number of qubits required and the Pauli strings generated
```
The sentence continues on the next line.